package.json

```diff
@@ -42,6 +42,7 @@
     "bcrypt": "^4.0.1",
     "class-transformer": "^0.2.3",
     "class-validator": "^0.11.1",
+    "express-rate-limit": "^5.1.1",
```

Comment:
This module, by default uses an in-memory store for tracking the rate limits - this means that if the environment running the application has multiple processes running, the limits can be bypassed to a certain extent. I'm not yet familiar enough to comment whether or not that would be a large issue for the project, my guess is that it wouldn't be a big deal. Alternatively it could be passed an external postgres store to mitigate that, as the state would be shared across processes/instances